Dockerfile

```diff
@@ -11,10 +11,14 @@ RUN apt-get update \
 ARG CUDA12_GENCODE='-gencode=arch=compute_90,code=sm_90'
```

Comment:
I can't comment on the commit message.

Please add Tested: to the commit message.

The test should at least be build cuda and udmabuf and see the build succeed, if possible, also do a neper run, but if that's too much out of your way it's not comptelely necessary.

Reply:
One more thing. 

I think having the build instructions be outside this repository is not ideal. It means every time the code changes, you need to update out-of-repo build instructions and keep them in sync.

If possible, please add README-devmem.md, and add the build instructions for CUDA and UDMABUF there, then replace the google docs where the instructions are with a link to the instructions in this github.

Reply:
Added a Tested tag